source/variables/breakpoints.css

```diff
@@ -0,0 +1,13 @@
+/* 
+The breakpoints for all resolutions represented as pairs with the breakpoint name and its associated pixel width.
+http://cssnext.io/features/#custom-media-queries 
+@media (--medium) { ..styles.. }
+*/
+:root {
+	@custom-media --breakpoint--extra-small (width <= 320px) and (width >= 0); /* Used for style that should not cascade out */
+	@custom-media --breakpoint--small  only screen and (min-width: 480px);
+	@custom-media --breakpoint--medium  only screen and (min-width: 768px);
+	@custom-media --breakpoint--large  only screen and (min-width: 1024px);
+	@custom-media --breakpoint--extra-large  only screen and (min-width: 1220px);
+	@custom-media --breakpoint--rotate (max-width: 480px, (context: 16))) and (orientation: portrait);
```

Comment:
Also I just tested and confirmed that general variables and @custom-media variables are isolated from one another, so there really is no need to have --breakpoint-- within the names these custom media queries. @media (XXXX) should be enough context already and we should never have to worry about say --small being used else where and conflicting.